control/veda_deliver_xuetang.py

```diff
@@ -8,7 +8,7 @@
 import json
```

Comment:
There's an upcoming PR deprecating this service, so don't sweat correctness here too much.